statsmodels/tsa/automatic/tests/test_sarimax.py

```diff
@@ -12,12 +12,23 @@
 macrodata = datasets.macrodata.load_pandas().data
 macrodata.index = pd.PeriodIndex(start='1959Q1', end='2009Q3', freq='Q')
 
+
 def test_non_stepwise():
-    """Auto order function for SARIMAX models."""
-    p, q = sarimax.auto_order(macrodata.infl, d=0)
-    # p, q = sarimax.auto_order(macrodata.infl, d=0, enforce_stationarity=False)
+    """test function for non-stepwise auto_order."""
+    p, q = sarimax.auto_order(macrodata['infl'], d=0)
+#    p, q = sarimax.auto_order(macrodata.infl, d=0, enforce_stationarity=False)
     desired_p = 2
     desired_q = 2
-    assert_equal(p, desired_p)
-    assert_equal(q, desired_q)
+    assert p == desired_p
```

Comment:
We use the numpy testing functions (regular `assert` calls can disappear at compile time), so something like:

```python
from numpy.testing import assert_equal
```

and then later:

```python
assert_equal(p, desired_p)
```

Reply:
Sure, I'll use the numpy testing functions.